src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -219,30 +226,29 @@ public SVCustomReadFilter(final UseEnd whichEnd,
                                   final boolean filterDupReads,
                                   final boolean filterSecondaryAlignments,
                                   final boolean filterSupplementaryAlignments,
-                                  final int MQThreshold){
+                                  final int MQThreshold,
+                                  final int mMQThreshold){
```

Comment:
it's pretty confusing to have MQThreshold and mMQThreshold. please spell out what these are, maybe: "mappingQuality", "mateMappingQuality"